src/scss/variables/_syntax.scss

```diff
@@ -0,0 +1,8 @@
+$drops--syntax: (
```

Comment:
as said in the previous review, we define here the syntax by default. But we want first the syntax to be defined by the `$trowel-config` variable. This variable exists here to allow the developer to overwrite for this component the syntax, if he wants to.

So we want here the variable $drops--syntax to be :

```scss
$drops--syntax: null !default;
```